lms/djangoapps/ccx/views.py

```diff
@@ -338,6 +339,31 @@ def get_ccx_for_coach(course, coach):
     return None
 
 
+def get_date(ccx, node, date_type=None, parent_node=None):
+    """
+    This returns override or master date for section, subsection or a unit.
+
+    :param ccx: ccx instance
+    :param node: chapter, subsection or unit
+    :param date_type: start or due
+    :param parent_node: parent of node
+    :return: start or due date
+    """
+    date = get_override_for_ccx(ccx, node, date_type, None)
+    master_date = node.start if date_type == "start" else node.due
+    if date:
```

Comment:
again, if you are checking that `date` is not `None` you should actually write it explicitly.